services/proxy/README.md

```diff
@@ -128,6 +128,14 @@ somewhat costly operation, especially if the user is a member of a large number
 groups. If the group memberships of a user are changed in the IDP after the
 first login, it can take up to 5 minutes until the changes are reflected in Infinite Scale.
 
+### Claim Updates
+
+OpenID Connect (OIDC) scopes are used by an application during authentication to authorize access to a user's detail, like name or picture. Each scope returns a set of user attributes, which are called claims. The scopes an application should request, depends on which user attributes the application needs. Once the user authorizes the requested scopes, the claims are returned in an ID Token.
```

Comment:
While the content is technically fine. I don't we should add this paragraph here. After all this is the Readme of the `proxy` service and not some general introduction to OpenID Connect.

Reply:
OICD is quite complex, and not everyone undestands this. Therefore I will add this paragraph to the admin docs anyways for the ease of a common understanding of terms and relations. If you think we should drop it here I can remove it.

Reply:
Yes, OIDC is complex. But still I believe the proxy's README is the wrong place for explaining it.